hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/net/Node.java

```diff
@@ -98,4 +98,12 @@ public interface Node {
    * @return true if this node is an ancestor of <i>n</i>
    */
   boolean isAncestor(Node n);
+
+  /**
+   * Judge if this node is under a specific scope.
```

Comment:
Judge if this node is a descendant of the node representing by the nodeLocation.